dist/COVID-VACCINE-TRIAL-schema.json

```diff
@@ -85,9 +88,6 @@
         "HEALTH_HISTORY::HIGH_BLOOD_PRESSURE": {
           "type": "boolean"
         },
-        "HEALTH_HISTORY::IMMUNOCOMPROMISED": {
```

Comment:
does order matter for this?  just curious

Reply:
The human label for "IMMUNOCOMPROMISED" is "Compromised immune system", so solving for for that. I know it isn't perfect — let me know if you'd like me to change the schema key to something like COMPROMISED_IMMUNE so ordering in the schema is alpha.

Reply:
Going to merge in b/c under a little bit of time crunch, but please reach out if you'd like to see the change made!